pkgs/swiftgen/swift2objc/lib/src/parser/parsers/parse_symbols_map.dart

```diff
@@ -0,0 +1,23 @@
+import '../../ast/declarations/built_in/built_in_declaration.dart';
+import '../_core/parsed_symbol.dart';
+import '../_core/utils.dart';
+
+final _builtInDeclarations = [
```

Comment:
Can this be `const`? I think you'd have to make `BuiltInDeclaration`'s constructor const, but it should work. I'm suspicious of any non-const static variables. Ffigen has a whole lot of static state, and it causes problems. I need to refactor it at some point. I can see you're not mutating this map at all, but it'd be good to enforce that if possible.